### DISCUSSION TITLE
Move errors from the bottom to the top

### DISCUSSION THREAD

**Discussion by jonocodes** [aminalaee/sqladmin#706] [Ideas]
In the create and edit templates, error messages under all the fields:
https://github.com/aminalaee/sqladmin/blob/0e132dadd8d07bcaa859159b8dd26666a8173c25/sqladmin/templates/edit.html#L29-L31

This is problematic if your model has many fields that extend beyond the bottom of the window. In that case there is no visible feedback of the error unless you happen to scroll to the bottom. 

So I think it would be better at the top.

Happy to make a PR if interested.

**Comment by aminalaee** [comment]
Thanks for the suggestion, isn't that like the standard way to show the form errors under each field or in front of it? I've never seen them to be shown above the field.

**Comment by jonocodes** [comment]
I am not talking about the errors per field. I am talking about the singular {{ error }} sent to the template. Perhaps for a server error or something. See the linked lines above.

**Reply by aminalaee** [reply]
Ah ok that makes sense, feel free to create a PR for that,

**Reply by jonocodes** [reply]
https://github.com/aminalaee/sqladmin/pull/707
